ansible/roles/install-olm/tasks/main.yml

```diff
@@ -0,0 +1,30 @@
+---
+- name: "Uninstall OLM"
+  ansible.builtin.command: "{{ op_bin_dir }}/{{ operator_sdk_binary }} olm uninstall --version {{ olm_version }} --timeout 5m0s"
+  changed_when: true
+  failed_when: false
+  register: olm_uninstall_rc
+  tags:
+    - clean
+
+- name: "Detect if OLM is present"
```

Comment:
- Please move detection before uninstall
- Check olm version
- If version is not what it requested force uninstall